### ISSUE TITLE
Deleting .ssh directory on self hosted runner is terrible

### ISSUE THREAD

**Issue by MikeWarriner** [shimataro/ssh-key-action#231]
Latest modifications to delete .ssh directory by default result in directory being deleted on self-hosted runner, removing remote access to server if any pre-existing SSH configuration such as authorized_hosts is present.

Please revoke change, or at least put behind a feature whose default value is to preserve the original settings.

**Comment by dpagini** [comment]
Ugh. This just caused me an enterprise outage as well. We are also using a self-hosted runner. I wasn't able to figure out what about this release caused everything to break, but rolling back to 2.4.0 fixed everything for me.

I was actually seeing that none of my SSH files were being copied to my runner when using the 2.5.0 build. So it wasn't just about the delete, I don't believe.

**Comment by mym0404** [comment]
👍🏻 I just restored all my local ssh keys......

I think the package should provide one of the following.
  - input to choose whether delete ssh dir or not
  - custom path to ssh install

**Comment by khoazero123** [comment]
I made fork for add new option `cleanup` to keep the `.ssh` folder.

https://github.com/khoazero123/ssh-key-action#usage

```
- name: Install SSH key for LIVE
        uses: khoazero123/ssh-key-action@dev
        with:
          key: ${{ secrets.DEPLOY_SSH_KEY_LIVE || vars.DEPLOY_SSH_KEY_LIVE }}
          name: id_rsa-bastion # optional
          known_hosts: unnecessary
          if_key_exists: replace # replace / ignore / fail; optional (defaults to fail)
          cleanup: false # Should remove .ssh directory? set false to keep .ssh directory
```

**Comment by albertodebortoli** [comment]
Thank you. I just tried this action and it destroyed some of our CI instances.

**Comment by albertodebortoli** [comment]
Thank you. I just tried this action and it caused an outage on our CI.

**Comment by shimataro** [comment]
Very sorry for the late reply.
This issue will be fixed in #251, which will be shipped soon.
